draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -796,14 +801,14 @@
    </t>
 
    <t>
-	To allow uCDN to use client-side caching of all triggers and trigger collections,  dCDN SHOULD include an ETag response header 
-	with every resource representation it sends. dCDN SHOULD also respond with the HTTP code 304 ("Not Modified") without a response 
-	body in response to conditional resource requests that use "If-None-Match" and/or "If-Modified-Since" request headers 
-	as described in Section 13 of
-	<xref target="RFC9110"/> , in case dCDN doesn't have a more recent resource representation. 
+	To allow uCDN to use client-side caching of all triggers and trigger collections,  each resource representation snet by dCDN 
```

Comment:
I think more changes are necessary if we want to head down the path of allowing Last-Modified.  Lines 4088 and 4133 represents text that likely needs to change as well.  Additionally, all the examples show ETags with no Last-Modified date.  We should likely add sensible Last-Modified dates to the examples to illustrate that you can use either.  Finally, we probably want to illustrate this with an example around the area of Lines 4093 and 4111.

Reply:
The examples now have Last-Modified and If-Modified-Since, and text that references ETag only